troubleshooting.md

```diff
@@ -49,13 +49,18 @@ The standard library headers are often found relative to the compiler.
 Try to configure your project with an absolute path to the compiler
 (`/usr/bin/gcc`, not `gcc`).
 The logs (with `--log=verbose`) can tell you the `cc1` command, including which
-paths were searched for the standard library. You can compare this to the output
+paths searched for the standard library. You can compare this to the output
```

Comment:
Is this change intentional? The previous wording looks correct.

Reply:
oops! will fix